modules/@apostrophecms/notification/ui/apos/components/TheAposNotifications.vue

```diff
@@ -29,31 +25,7 @@ export default {
     };
   },
   async mounted() {
-    apos.notify = async function(message, options) {
```

Comment:
We had support for %s placeholders, but haven't documented it in the 3.x era, and i18next approaches interpolation in a different way. Decided to gut this, simplifying the code. Worst case a few people with custom 2.x style notifications using interpolation (which is pretty rare) will see some %s placeholders and switch them over to `{{ }}`. I think this early in 3.x we can do this and avoid a lot of redundant code that would require maintenance.

Reply:
apos.notify, apos.confirm, and apos.alert all support an "interpolate" option which may contain an object of key-value pairs to be inserted using `{{ key }}`, using `$t`. They also support a `localize: false` option for situations where the message has already been localized, or is in some cases, in which case `$t` is used directly for some of the inputs.

While i18next's `$t` handles interpolation keys just by chucking them in the same options object with options that have a special impact on the function, and I can do nothing about that, I didn't want to continue that madness into our own APIs in an unmaintainable way.